src/mme-app/actionHandlers/serviceRequestActionHandlers.cpp

```diff
@@ -399,14 +399,11 @@ ActStatus ActionHandlers::process_mb_resp_svc_req(ControlBlock &cb)
     }
 
     MsgBuffer *msgBuf = static_cast<MsgBuffer*>(cb.getMsgData());
+    VERIFY(msgBuf, return ActStatus::ABORT, "Invalid message buffer \n");
+    VERIFY(msgBuf->getLength() >= sizeof(struct MB_resp_Q_msg), return ActStatus::ABORT, "Invalid message length \n");
```

Comment:
In what case would we hit this scenario ?

Reply:
This is just a safety check to ensure that the message has minimum length, before dereferencing the fields.